configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/posetrack18/hrnet_w32_posetrack18_384x288.py

```diff
@@ -110,7 +110,7 @@
         type='NormalizeTensor',
         mean=[0.485, 0.456, 0.406],
         std=[0.229, 0.224, 0.225]),
-    dict(type='TopDownGenerateTarget', sigma=2),
+    dict(type='TopDownGenerateTarget', sigma=3),
```

Comment:
Why are these 2 configs modified?

Reply:
The input size 384x288 is usually with `sigmas = 3`. So here I change it. 

But I do not retrain these two configs and do not update the results on the `hrnet_posetrack18.md`.

Reply:
We'd better not change the config before it has been verified. Users may not be able to get the reported performance with new configs.

Reply:
Also, it's irrelevant to this PR. We can update the configs in a separate PR.

Reply:
Got it. Changed back to these two configs.